test/test_artifacts/autogluon.test.Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG COSMOS_IMAGE
+FROM $COSMOS_IMAGE as base
+
+ARG MAMBA_DOCKERFILE_ACTIVATE=1
+ARG AUTOGLUON_VERSION="0.8.2"
+
+RUN sudo apt-get update && sudo apt-get install -y git graphviz graphviz-dev
+
+# TODO: remove autogluon once it's included into the distribution
+RUN micromamba install -y conda-forge::pytest conda-forge::pygraphviz conda-forge::jupyter conda-forge::nbconvert conda-forge::autogluon==$AUTOGLUON_VERSION
```

Comment:
Can you try the same command with --freeze-insalled flag

Reply:
sure. Let me add that

Reply:
So we have some conflicts here:
```bash
error    libmamba Could not solve for environment specs
    The following packages are incompatible
    ├─ autogluon 0.8.2  is not installable because it requires
    │  └─ autogluon.core 0.8.2 , which requires
    │     └─ grpcio >=1.42.0,<=1.50.0 , which conflicts with any installable versions previously reported;
    ├─ grpcio is not installable because it conflicts with any installable versions previously reported;
    └─ nbconvert is not installable because it conflicts with any installable versions previously reported.
Possible hints:
  - 'freeze_installed' is turned on
```

Reply:
grpcio pin is needed as there was some issue with the newer version which would break multimodal functionality

Reply:
Regarding nbconvert, it probably already exists in the environment and I can remove that. It is only used for testing

Reply:
Can you relax the constraints on grpcio then (in autogluon.core)?

This is because our latest image version 0.4.2 , requires 1.54.3 but you have a constraint of >=1.42.0,<=1.50.0